utils/firestore.js

```diff
@@ -1381,8 +1381,10 @@ const submitSpecimen = async (biospecimenData, participantData, siteTubesList) =
             .where('state.uid', '==', participantUid)
             .get();
         const eligibilityUpdates = processMouthwashEligibility(participantSnapshot.docs[0].data());
-        
-        await participantSnapshot.docs[0].ref.update(eligibilityUpdates);
+
+        if(eligibilityUpdates && Object.keys(eligibilityUpdates).length) {
```

Comment:
```suggestion
        if (eligibilityUpdates && Object.keys(eligibilityUpdates).length) {
```

Nit: adding space after if